packages/docs/community/release-process/smart-contracts.md

```diff
@@ -34,26 +29,53 @@ Ongoing smart contract development is done on the `master` branch.
 ### Github tags
 All release branches should be tagged as such, e.g. `celo-contracts-N`. Each should include a summary of the release contents.
 
-## Build process
-A new release can be built by running the following script:
+### Contracts
+Every deployed smart contract has its current version number as a constant which is publicly accessible via the `getVersion()` function, which returns the storage, major, minor, and patch version. Version number is encoded in the Solidity source and updated as part of code changes.
+
+Contracts deployed to a live network without the `getVersion()` function, such as the original set of core contracts, are to be considered version `1.0.0.0`.
+
+## Build and Release Process
+Using the following scripts, new contracts can be built and deployed alongside a corresponding on-chain governance proposal. Run the following scripts from the branch with the new release contracts:
 
 ```bash
-yarn run release-contracts -d $CURRENTLY_RELEASED_BRANCH -f $FROM_ADDRESS -n $NETWORK
-```
+## Script 1 ##
+# verify the current contract bytecode on the network matches the contracts currently on the RC1 branch
+yarn truffle exec --network $NETWORK ./scripts/truffle/verify-bytecode.js --build_artifacts build/rc1 --before_release_1
+
+## Script 2 ##
+# generate report on backwards compatibility between RC1 and current branch with new release contracts
+yarn ts-node scripts/check-backward.ts report --exclude ".*Test|Mock.*|I[A-Z].*|.*Proxy|LinkedList|SortedLinkedList|SortedLinkedListWithMedian|MultiSig.*|ReleaseGold" -o build/rc1 -n build/contracts --output_file report.json
 
-This script does the following:
+## Script 3 ##
+# deploy new contracts and generate governance proposal for upgrade
+yarn truffle exec --network development ./scripts/truffle/make-release.js --build_directory build/ --report report.json --proposal proposal.json --initialize_data example-initialize-data.json'      
```

Comment:
Ditto, and we actually have the simplified command already

Reply:
ok maybe you or @yorhodes can help me figure out what it is.

Reply:
Ditto about follow-up PR